k2/python/tests/fsa_test.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+#
+# Copyright (c)  2020  Mobvoi Inc.        (authors: Fangjun Kuang)
+#
+# See ../../../LICENSE for clarification regarding multiple authors
+
+# To run this single test, use
+#
+#  ctest --verbose -R fsa_test_py -E host
+
+import unittest
+
+import k2
+import torch
+
+
+class TestFsa(unittest.TestCase):
+
+    def test_acceptor_from_str(self):
+        s = '''0 1 2 -1.2
+            0 2  10 -2.2
+            1 3  3  -3.2
+            1 6 -1  -4.2
+            2 6 -1  -5.2
+            2 4  2  -6.2
+            3 6 -1  -7.2
+            5 0  1  -8.2
+            6
+        '''
+
+        fsa, aux_labels = k2.fsa_from_str(s)
+        assert aux_labels is None
+
+        expected_str = '''0 1 2 -1.2
+0 2 10 -2.2
```

Comment:
strange format here..

Reply:
This is equivalent to
```
0 1 2 -1.2
0 2 10 -2.2
```

It has no leading spaces.

Reply:
If you can think of a better-looking format, I am willing to do a switch.

Reply:
it's OK, not a big issue, just feel  aligned columns  would be easy to read.

Reply:
Fixed.